src/SmartComponents/DriftPage/ExportButton/ExportButton.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { compareActions } from '../../modules';
+import { DownloadIcon } from '@patternfly/react-icons';
+import { Button } from '@patternfly/react-core';
+
+class ExportButton extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <Button
```

Comment:
Can we set this to `float: right` so it is on the right side of the grid element?